controller/src/main/java/io/pravega/controller/util/Config.java

```diff
@@ -177,6 +177,15 @@
     public static final Property<Integer> PROPERTY_RETENTION_THREAD_COUNT = Property.named(
             "retention.thread.count", 1, "retention.threadCount");
 
+    public static final Property<Long> PROPERTY_TXN_GC_FREQUENCY_MINUTES = Property.named(
+            "retention.frequency.minutes", 60L, "transaction.gc.frequencyMinutes");
```

Comment:
Can this be defined in seconds? Being an integer in minutes precludes values less than 1, which will be needed for flink.

Reply:
I'm not sure I understand this. Why would flink require txn GC to happen at a rate faster that 1 min?

Reply:
Yes. It is very normal to have this in seconds. 1-10 seconds being the most common configuration. We may even get a request for it in millis, but we can probably push back a bit on that.